packages/swarm-docs/src/pages/Button.mdx

```diff
@@ -23,9 +22,56 @@ Buttons are the primary way to interact with content.
 ## Props
 <PropsTable propMetaData={Button.__docgenInfo} />
 
+## Anatomy
+All button styles and sizes share the same anatomy.
+<img src="/static/docs-images/button-anatomy.png" width="810" height="252" />
 
-## a11y
+## Styles
+
+** Default **
+
+<img src="/static/docs-images/button-secondary-states.png" width="810" height="132" />
+<br/>
+
+** Primary **
+
+<img src="/static/docs-images/button-primary-states.png" width="810" height="132" />
+<br/>
+
+** Neutral **
+
+<img src="/static/docs-images/button-neutral-states.png" width="810" height="132" />
+<br/>
+
+** Bordered **
+
+<img src="/static/docs-images/button-bordered-states.png" width="810" height="132" />
+<br/>
+
+** Disabled **
 
+<img src="/static/docs-images/button-disabled-states.png" width="250" height="132" />
+
+
+
```

Comment:
nitpick, are these extra newlines necessary? Looks like [Prettier got MDX support](https://github.com/prettier/prettier/pull/4975) late last year, so we might be able to auto-format some of this stuff cc @hsbacot

Reply:
just an oversight, I'll clean up